experiments/functional/zfs-LocalPV/zv-properties-verify/test.yml

```diff
@@ -0,0 +1,150 @@
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+    
+  tasks:
+    - block:
+
+        ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'
+        
+        - block:
+
+          - name: Get the zvolume name
+            shell: >
+              kubectl get pvc {{ pvc_name }} -n {{ app_ns }} --no-headers
+              -o custom-columns=:.spec.volumeName
+            args: 
+              executable: /bin/bash
+            register: zvol_name
+
+          - name: Record the zvolume name
+            set_fact:
+              zv_name: "{{ zvol_name.stdout }}"
+
+          - name: Get the storage class name used to create volume
+            shell: >
+              kubectl get pvc {{ pvc_name }} -n {{ app_ns }} --no-headers
+              -o custom-columns=:.spec.storageClassName
+            args: 
+              executable: /bin/bash
+            register: stg_class_name
+
+          - name: Record the storage class name
+            set_fact:
+              sc_name: "{{ stg_class_name.stdout }}"
+
+          - name: Get the value of compression parameter from the storage class
+            shell: >
+              kubectl get sc {{ sc_name }} --no-headers 
+              -o custom-columns=:.parameters.compression
+            args:
+              executable: /bin/bash
+            register: compression_parameter
+
+          - name: Compare this value with the compression field in zvolume
+            shell: > 
+              kubectl get zv {{ zv_name }} -n {{ operator_ns }} --no-headers
+              -o custom-columns=:.spec.compression
+            args: 
+              executable: /bin/bash
+            register: result
+            failed_when: compression_parameter.stdout != result.stdout
+            
+          - name: Get the value of dedup parameter from the storage class
+            shell: >
+              kubectl get sc {{ sc_name }} --no-headers
+              -o custom-columns=:.parameters.dedup
+            args:
+              executable: /bin/bash
+            register: dedup_parameter
+
+          - name: Compare this value with the dedup field in zvolume
+            shell: > 
+              kubectl get zv {{ zv_name }} -n {{ operator_ns }} --no-headers
+              -o custom-columns=:.spec.dedup
+            args: 
+              executable: /bin/bash
+            register: result
+            failed_when: dedup_parameter.stdout != result.stdout
+
+          - name: Get the file system type from the storage class on top of which application is deployed
```

Comment:
Get the file system type to be created on application mount from the storage class

Reply:
Done !